test/e2e/common/pods.go

```diff
@@ -175,9 +178,11 @@ func expectNoErrorWithRetries(fn func() error, maxRetries int, explain ...interf
 
 var _ = framework.KubeDescribe("Pods", func() {
 	f := framework.NewDefaultFramework("pods")
+	var dc dynamic.Interface
 	var podClient *framework.PodClient
 	ginkgo.BeforeEach(func() {
 		podClient = f.PodClient()
+		dc = f.DynamicClient
```

Comment:
Why not `dc := f.DynamicClient` in the one test that uses this?

Reply:
This was a pattern which appears in a few files. Will make things more extensible in the future, however totally fine with moving it into the test itself if that's what is better

Reply:
ok I'm fine with it if there's prior art